integrations/connext/components/transfer.tsx

```diff
@@ -0,0 +1,168 @@
+            />
+            <span className="3xl:text-xl text-xs font-medium">{originChain?.name}</span>
+          </div>
+          <div className="flex items-center justify-end space-x-1.5">
+            <Image
+              alt={`${destinationChain?.name || 'Chain'} logo`}
+              className="3xl:w-6 3xl:h-6 rounded-full"
+              height={20}
+              src={destinationChain?.image || '/integrations/connext/logos/chains/gnosis.png'}
+              width={20}
+            />
+            <span className="3xl:text-xl text-xs font-medium">{destinationChain?.name}</span>
+          </div>
+        </div>
+        <div className="my-4 flex items-start justify-between space-x-2">
+          <div className="flex flex-col space-y-1.5">
+            <span className="3xl:text-xl text-sm">{calculateAmount(transfer.origin_transacting_amount, originDecimals || 18)}</span>
+            <div className="flex items-center justify-start space-x-1">
+              <Image
+                alt={`${originAsset?.name || ''}`}
+                className="3xl:w-5 3xl:h-5 rounded-full"
+                height={16}
+                src={originAsset?.image || '/integrations/connext/logos/chains/gnosis.png'}
+                width={16}
+              />
+              <span className="3xl:text-xl text-xs font-medium">{originAsset?.symbol}</span>
+            </div>
+          </div>
+          <div className="flex flex-col items-center">
+            <a
+              href={`https://${isMainnet ? 'connextscan.io' : 'testnet.connextscan.io'}/tx/${transfer.transfer_id}`}
+              rel="noopener noreferrer"
+              target="_blank">
+              {getStatusIcon(transfer.status || '', transfer.error_status || '')}
+            </a>
+          </div>
+          <div className="flex flex-col items-end space-y-1.5">
+            <span className="3xl:text-xl text-sm">
+              {calculateAmount(transfer.destination_transacting_amount ?? transfer.origin_transacting_amount, destinationDecimals || 18)}
+            </span>
+            <div className="flex items-center justify-center space-x-1">
+              <Image
+                alt={`${destinationAsset?.name || ''}`}
+                className="3xl:w-5 3xl:h-5 rounded-full"
+                height={16}
+                src={destinationAsset?.image || '/integrations/connext/logos/chains/gnosis.png'}
+                width={16}
+              />
+              <span className="3xl:text-xl text-xs font-medium">{destinationAsset?.symbol}</span>
+            </div>
+          </div>
+        </div>
+        <div className="mt-0.5 flex items-center justify-between">
+          <span></span>
+          <div className="flex items-center">
+            <span className="3xl:text-xl text-xs font-medium text-slate-400 dark:text-slate-500">{formatTimestamp(transfer.xcall_timestamp)}</span>
+          </div>
+        </div>
+        <div className="mt-1 -mb-2 flex items-center justify-end">
+          <a
```

Comment:
use the [LinkComponent](https://github.com/turbo-eth/template-web3-app/blob/integrations/components/shared/link-component.tsx) rather than an anchor tag